neqo-http3/src/connection.rs

```diff
@@ -940,36 +939,6 @@ impl Http3Connection {
         Ok(())
     }
 
-    /// Set the stream `SendOrder`.
-    ///
-    /// # Errors
-    ///
-    /// Returns `InvalidStreamId` if the stream id doesn't exist
-    pub fn stream_set_sendorder(
```

Comment:
Are you *sure* that Firefox doesn't use this?  Because WebTransport is supposed to use these functions.